src/net/java/sip/communicator/impl/gui/main/call/DesktopSharingMouseAndKeyboardListener.java

```diff
@@ -244,6 +250,7 @@ public void remoteControlGranted(RemoteControlGrantedEvent event)
         {
             if(videoComponent != null)
             {
+                DTMFHandler.enableSendingDTMFTone(false);
```

Comment:
You can enable/disable the DTMFHandler by adding methods to CallPeerRenderer and get the associated renderer via
`CallManager.getActiveCallContainer(getCallPeer().getCall()).getCurrentCallRenderer().getCallPeerRenderer(getCallPeer())` or you can pass it down into the constructor of `DesktopSharingMouseAndKeyboardListener`.